docs/includes/grpc-marshalling.adoc

```diff
@@ -0,0 +1,107 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019, 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/..]
+:description: Helidon gRPC Marshalling
+:keywords: helidon, grpc, java, marshalling
+
+= Marshalling
+
+== Default Marshalling Support
+
+Helidon gRPC supports Protobuf out of the box. Protobuf marshaller will be used by default for any request and response classes that extend `com.google.protobuf.MessageLite` class, which is the case for all classes generated from a `proto` file using `protoc` compiler.
+
+That means that you don't need any special handling or configuration in order to support Protobuf serialization of requests and responses.
+
+== Custom Marshalling
+
+Helidon makes the use of custom marshallers trivial and provides one custom implementation, link:{helidon-github-tree-url}/grpc/core/src/main/java/io/helidon/grpc/core/JsonbMarshaller.java[JSON-B Marshaller], out of the box.
```

Comment:
In the link text, "JSON-B Marshaller" -> "JSON-B marshaller" or change it to "`JsonbMarshaller`"